monai/bundle/scripts.py

```diff
@@ -600,11 +600,15 @@ def download(
             _download_from_github(repo=repo_, download_path=bundle_dir_, filename=name_ver, progress=progress_)
         elif source_ == "monaihosting":
             try:
+                extract_path = os.path.join(bundle_dir_, name_)
```

Comment:
Would it make more sense if we use the huggingface cache path, instead of the MONAI cache path?

Reply:
Hi @mingxin-zheng , I followed: https://github.com/Project-MONAI/MONAI/pull/8403/files#diff-a715c9207f1e727d7a6ed7c6b25dfb57045d4429fc0b4eed3f5396e26ab34f80L630
maybe we can update to hf cache path in next version, what do you think?

Reply:
I guess this can stay open to discussion. Maybe we should hear more from the users. I'll closing the comment here